### ISSUE TITLE
Exit code should be non-zero if push fails

### ISSUE THREAD

**Issue by wulftone** [MaartenDesnouck/google-apps-script#81]
Simple example:
```bash
$ gas push && notify-send 'success' || notify-send 'fail'
Pushing 'My Project' to Google Drive... [✘]
Google doesn't accept the code you are trying to push. Try doublechecking for project structure errors. Push was reverted. [✘]
$ echo $?
0
```
Always notifies "success".

**Comment by MaartenDesnouck** [comment]
Jup. Thought about that, should be fixed indeed.

**Comment by MaartenDesnouck** [comment]
Released in 2.5.0
